scripts/python/tao_build.py

```diff
@@ -688,7 +688,22 @@ def add_to_tar(tar, file, dir_in_tar="", name_in_tar=""):
             add_to_tar(tar, build_info_file)
             if libstdcxx_path:
                 add_to_tar(tar, libstdcxx_path, name_in_tar=libstdcxx_name)
-        logger.info("sdk package created   : " + dsw_tgz)
+        logger.info("sdk package created : " + dsw_tgz)
+
+        # pkg for tf serving
+        serving_tgz = "{}/tao/tao_serving_sdk_{}.tgz".format(root, args.version)
+        python_bin = os.path.join(args.venv_dir, "bin", "python3")
+        tf_major, _, _, _, _, _, _, _ = get_tf_info(python_bin)
+        # patchelf to remove libtensorflow_framework.so.* deps
+        patchelf_cmd = f"patchelf --remove-needed libtensorflow_framework.so.{tf_major} {F_TAO_OPS_SO}"
```

Comment:
May have unwanted side-effect that the shared library will be left unusable for cases other than tfserving... consider making a copy?

Reply:
change to use `genrule` to generate so file in different dir.